release/pipeline/configs/environment_config.py

```diff
@@ -15,22 +15,25 @@
 
 EMAIL_LIST = ['istio-release@google.com']
 
-#Make the release on the 3ed Thursday of the mounth.
+#Make the release on the 3rd Thursday of the month.
 #It will run on at 17:15 GST
 MONTHLY_RELEASE_TRIGGER = '15 17 * * 4#3'
 
 AIRFLOW_CONFIG = dict(
+    # variable inputs
+    MFEST_COMMIT='{branch}@{{{timestamp}}}',
+    VERSION='{branch}-{date}-{rc}',
+    GCS_DAILY_PATH='daily-build/{VERSION}',
+    GCS_MONTHLY_STAGE_PATH='prerelease/{VERSION}',
+    GCS_MONTHLY_RELEASE_PATH='istio-release/releases/{VERSION}',
```

Comment:
There should be one set of paths, instead of having separate for daily and monthly.

And the param values can be different based on the context (i.e. monthly, daily, weekly, etc).